### PR TITLE
Add Small image ui style tests

### PR DESCRIPTION
**Pull request by rymorale**
## Description
These tests should be merged after https://github.com/adobe/aepsdk-messaging-android/pull/308

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+            .textColumnStyle(textColumnStyle)
+            .build()
+
+        // verify image style parameters
+        assertEquals(imageStyle.alpha, style.imageStyle.alpha)
+        assertEquals(imageStyle.colorFilter, style.imageStyle.colorFilter)
+        assertEquals(imageStyle.contentDescription, style.imageStyle.contentDescription)
+        assertEquals(imageStyle.alignment, style.imageStyle.alignment)
+        assertEquals(imageStyle.contentScale, style.imageStyle.contentScale)
+        assertEquals(imageStyle.modifier, style.imageStyle.modifier)
```
Comment (prudrabhat) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt:125]: Can you add helper methods for comparing style totally similar elements. That will allow you to re-use them for other templates  and make these tests less verbose (thought partial style overrides will still need to compared individually)

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        assertEquals(rootRowStyle.modifier, style.rootRowStyle.modifier)
+
+        // verify text column style parameters
+        assertEquals(textColumnStyle.modifier, style.textColumnStyle.modifier)
+        assertEquals(textColumnStyle.horizontalAlignment, style.textColumnStyle.horizontalAlignment)
+        assertEquals(textColumnStyle.verticalArrangement, style.textColumnStyle.verticalArrangement)
+    }
+
+    @Test
+    fun `create SmallImageUIStyle with some custom builder styles used`() {
```
[the remark this answers is not in the document]
Comment (rymorale) [reply code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt:193]: sure, i'll get these split up per style type

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/AepButtonStyleTests.kt [hunk 3]
```diff
@@ -57,6 +59,12 @@ class AepButtonStyleTests {
         assertEquals(overridingStyle.border, result.border)
         assertEquals(overridingStyle.colors, result.colors)
         assertEquals(overridingStyle.contentPadding, result.contentPadding)
+        assertEquals(overridingStyle.textStyle?.textStyle, result.textStyle?.textStyle)
+        assertEquals(overridingStyle.textStyle?.overflow, result.textStyle?.overflow)
+        assertEquals(overridingStyle.textStyle?.softWrap, result.textStyle?.softWrap)
+        assertEquals(overridingStyle.textStyle?.maxLines, result.textStyle?.maxLines)
+        assertEquals(overridingStyle.textStyle?.minLines, result.textStyle?.minLines)
+        assertEquals(overridingStyle.textStyle?.modifier, result.textStyle?.modifier)
```
Comment (spoorthipujariadobe) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/AepButtonStyleTests.kt:67]: nit: Can we use the new utility `validateTextStyle` here?

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mockito.Mockito.mock
+
+class SmallImageUIStyleTests {
+    val defaultSmallImageUIStyle = SmallImageUIStyle.Builder().build()
```
Comment (spoorthipujariadobe) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt:35]: nit: Can we also add a test for `create SmallImageUIStyle no builder styles used` and verify that the default style is used to all elements?

**Review by spoorthipujariadobe — APPROVED** [review]
LGTM, some minor nits

**Pull request merged**